NotifyMe/utils/websocket_utils.py

```diff
@@ -0,0 +1,87 @@
+from channels.layers import get_channel_layer
+from asgiref.sync import async_to_sync
+
+
+class NotificationManager:
+  
+  def maintenance_alert(self, group_name, message):
+    channel_layer = get_channel_layer()
+    async_to_sync(channel_layer.group_send)(
+      group_name,
+      {
+        "type":"send_maintenance_alert",
+        "message":message,
+        "notification_type":"maintenance_alert"
+      }
+    )
+    
+class CustomException(Exception):
```

Comment:
it should be in separate file since you extending from exception  then handle_exception should take care of the logging and raising that customException but here you are returning and handle_api_exception should take minimum two parameter one is error_code,message and other might be exception object and status_code